docs/modules/release-notes/pages/5-3-3.adoc

```diff
@@ -0,0 +1,20 @@
+= 5.3.3 Release Notes
+
+== Enhancements
+
+* Changed the exception type from `CancellationException` to `CancellationByUserException` in case the user cancels a job before it is initialized.
+https://github.com/hazelcast/hazelcast/pull/25452[#25452]
+* Updated the gRPC dependency version to 1.57.0.
+https://github.com/hazelcast/hazelcast/pull/25430[#25430]
+* Renamed the service port for Hazelcast clusters deployed in Kubernetes environments as `hazelcast`.
+Previously, the name was `hazelcast-service-port` causing the member auto-discovery (for embedded deployments) to fail.
+https://github.com/hazelcast/hazelcast/pull/24834[#24834]
+
+== Fixes
```

Comment:
This PR is notable from my POV https://github.com/hazelcast/hazelcast/pull/25661

Reply:
and other permission related improvements in https://github.com/hazelcast/hazelcast/pull/25539